DNN Platform/Library/Entities/Host/Host.cs

```diff
@@ -602,6 +602,27 @@ public static bool EnableSMTPSSL
             }
         }
 
+        /// -----------------------------------------------------------------------------
+        /// <summary>
+        ///   Gets a value indicating the SMTP OAuth provider.

```

Comment:
```suggestion
        ///   returns the currently configured SMTP OAuth provider if existing, for the current portal if portal SMTP enabled, otherwise for the installation

```